Formula/lrzsz.rb

```diff
@@ -62,6 +62,7 @@ def install
 
     system "make", "install"
     bin.install_symlink "lrz" => "rz", "lsz" => "sz"
+    bin.install_symlink "lsz" => "sb"
```

Comment:
Why provide a symlink with a different name, if it is not provided by upstream?

Reply:
Linux distros such as Debian provide these (sz, sb...) symlinks e.g. https://packages.debian.org/bullseye/amd64/lrzsz/filelist, so we'll need to check why they are not created by default by upstream.

Reply:
Looking at `Specfile.in` upstream and it apparently does something like `strip $RPM_BUILD_ROOT/usr/bin/{rb,rx,rz,sb,sx,sz}` which means it does create these symlinks for RPM packaging.

Reply:
Upstream build neither creates the sb symlink, nor the sz symlink created by this recipe already.

Reply:
Nudge to keep PR open.

Reply:
> Why provide a symlink with a different name, if it is not provided by upstream?

One symlink, sz, is created by this recipe already. sb program name makes lrzsz use the ymodem protocol. I don't know why upstream chose not to create these symlinks as part of its build.